src/events/delegate.js

```diff
@@ -3,7 +3,7 @@ import bind from "./bind.js";
 
 function delegate (subject, type, selector, callback) {
 	bind(subject, type, function(evt) {
-		if (evt.target.closest(selector)) {
+		if ((evt.target.closest ? evt.target : evt.target.parentNode).closest(selector)) {
```

Comment:
What is this change for? Any `Element` instance has `closest`, so I’m not sure what `EventTarget` instances there are that don’t have `closest` but their parents do. Even if such a node exists, this should be documented with comments because it's totally non-obvious.